components/XOGame.js

```diff
@@ -4,12 +4,19 @@ import { useEffect, useState } from "react";
 const players = {
   CPU: {
     SYMBOL: "O",
+    NAME: "Computer",
   },
   HUMAN: {
     SYMBOL: "X",
+    NAME: "You",
   },
 };
 
+const playerSymbol = players?.HUMAN?.SYMBOL;
+const cpuSymbol = players?.CPU?.SYMBOL;
+const playerName = players?.HUMAN?.NAME;
+const cpuName = players?.CPU?.NAME;
```

Comment:
Question marks are not needed here since you just defined the object with all necessary properties beforehand.

```suggestion
const playerSymbol = players.HUMAN.SYMBOL;
const cpuSymbol = players.CPU.SYMBOL;
const playerName = players.HUMAN.NAME;
const cpuName = players.CPU.NAME;
```

Reply:
Ok i have all changed :)